mcv/hostname.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+
+def _read_stripped_lines(f):
+    return [l.rstrip() for l in f.readlines()]
+
+
+def _splice_alias(alias, line):
```

Comment:
That's technically true, but this module is just a helper for setting the current systems hostname, which presumably only affects localhost